core/web/api/templates/investigation_api.html

```diff
@@ -8,16 +8,26 @@
     <th>Name</th>
     <th>Created</th>
     <th>Updated</th>
+    <th>Created By</th>
     <th>Nodes</th>
   </tr>
   {% for obj in data %}
   <tr class="node-line">
     <td class="icon-cell">
         <a href="{{ url_for('frontend.InvestigationView:graph', id=obj['id']) }}"><i class="flaticon-network38"></i></a>
     </td>
-    <td><a href="{{ url_for("frontend.InvestigationView:get", id=obj['id'])}}">{{obj['name']}}</a></td>
+    <td>
+      <a href="{{ url_for("frontend.InvestigationView:get", id=obj['id'])}}">
+        {% if obj['name'] %}
+          {{obj['name']}}
+        {% else %}
+          Unnamed
+        {% endif %}
+      </a>
+    </td>
     <td>{{macros.display_datetime(obj.created)}}</td>
     <td>{{macros.display_datetime(obj.updated)}}</td>
+    <td>{% if obj.created_by %}{{obj.created_by}}{% endif %}</td>
```

Comment:
What about something like `{{obj.created_by or ''}}`. This allows you to remove the `{%if%}` clause